assets/blocks/quiz/quiz-block/use-has-questions.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+import { select, dispatch } from '@wordpress/data';
+
+/**
+ * Monitor for questions and disable the lesson quiz when none have been added.
+ *
+ * @param {string} clientId The quiz block client id.
+ */
+export const useHasQuestions = ( clientId ) => {
+	const questionBlocks = select( 'core/block-editor' )
+		.getBlocks( clientId )
+		.filter( ( block ) => !! block.attributes.title );
```

Comment:
I think this can be wrapped in a `useSelect` for some gains.

Reply:
Done in 904a6694a5bd4900419211b3b49b7083810c0b59